tests/ProductTest.php

```diff
@@ -1464,6 +1464,47 @@ public function testProcessVariationSpecificProperties($previouslyParsedData, $d
         $this->assertSame($expectedResult, $productMock->getField('attributes'));
     }
 
+    /**
+     * @test
+     */
+    public function testProcessVariationSpecificPropertiesReturnsWithoutProcessingIfPropertiesRegistryReturnsFalse()
+    {
+        $registryMock = $this->getMockBuilder(\Findologic\Plentymarkets\Registry::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['get'])
+            ->getMock();
+        $registryMock->expects($this->once())->method('get')->will($this->returnValue(false));
```

Comment:
Replace `->will($this->returnValue(false))` with `->willReturn(false)`. This will save us one method call.